diagram.json

```
@@ -0,0 +1,157 @@
+      { "type": "wokwi-neopixel", "id": "rgb6", "top": -22.7, "left": -97, "attrs": {} },
+      { "type": "wokwi-neopixel", "id": "rgb7", "top": -22.7, "left": -154.6, "attrs": {} },
+      { "type": "wokwi-neopixel", "id": "rgb8", "top": -22.7, "left": -212.2, "attrs": {} },
+      { "type": "wokwi-neopixel", "id": "rgb9", "top": -22.7, "left": -337, "attrs": {} },
+      { "type": "wokwi-neopixel", "id": "rgb10", "top": -22.7, "left": -279.4, "attrs": {} },
+      { "type": "wokwi-neopixel", "id": "rgb11", "top": -128.3, "left": -97, "attrs": {} },
+      { "type": "wokwi-neopixel", "id": "rgb12", "top": -128.3, "left": -154.6, "attrs": {} },
+      { "type": "wokwi-neopixel", "id": "rgb13", "top": -128.3, "left": -212.2, "attrs": {} },
+      { "type": "wokwi-neopixel", "id": "rgb14", "top": -128.3, "left": -279.4, "attrs": {} },
+      { "type": "wokwi-neopixel", "id": "rgb15", "top": -128.3, "left": -337, "attrs": {} },
+      { "type": "wokwi-neopixel", "id": "rgb16", "top": -224.3, "left": -337, "attrs": {} },
+      { "type": "wokwi-neopixel", "id": "rgb17", "top": -224.3, "left": -279.4, "attrs": {} },
+      { "type": "wokwi-neopixel", "id": "rgb18", "top": -224.3, "left": -212.2, "attrs": {} },
+      { "type": "wokwi-neopixel", "id": "rgb19", "top": -224.3, "left": -154.6, "attrs": {} },
+      { "type": "wokwi-neopixel", "id": "rgb20", "top": -224.3, "left": -97, "attrs": {} },
+      { "type": "wokwi-neopixel", "id": "rgb21", "top": -301.1, "left": -97, "attrs": {} },
+      { "type": "wokwi-neopixel", "id": "rgb22", "top": -301.1, "left": -154.6, "attrs": {} },
+      { "type": "wokwi-neopixel", "id": "rgb23", "top": -301.1, "left": -212.2, "attrs": {} },
+      { "type": "wokwi-neopixel", "id": "rgb24", "top": -301.1, "left": -279.4, "attrs": {} },
+      { "type": "wokwi-neopixel", "id": "rgb25", "top": -301.1, "left": -346.6, "attrs": {} },
+      { "type": "wokwi-vcc", "id": "vcc2", "top": -95.24, "left": -374.4, "attrs": {} },
+      { "type": "wokwi-gnd", "id": "gnd2", "top": 28.8, "left": -375, "attrs": {} },
+      { "type": "wokwi-vcc", "id": "vcc3", "top": -172.04, "left": -48, "attrs": {} },
+      { "type": "wokwi-gnd", "id": "gnd3", "top": -96, "left": -48.6, "attrs": {} },
+      { "type": "wokwi-gnd", "id": "gnd4", "top": -172.8, "left": -384.6, "attrs": {} },
+      { "type": "wokwi-vcc", "id": "vcc4", "top": -277.64, "left": -384, "attrs": {} },
+      { "type": "wokwi-vcc", "id": "vcc5", "top": -364.04, "left": -48, "attrs": {} },
+      { "type": "wokwi-gnd", "id": "gnd5", "top": -268.8, "left": -48.6, "attrs": {} },
+      { "type": "wokwi-membrane-keypad", "id": "keypad1", "top": -578, "left": 72.8, "attrs": {} },
+      {
+        "type": "wokwi-buzzer",
+        "id": "bz1",
+        "top": -93.6,
+        "left": 501,
+        "attrs": { "volume": "0.1" }
+      }
+    ],
+    "connections": [
+      [ "pico:GP0", "keypad1:C4", "green", [ "h0" ] ],
```

Os pinos usados tem que ser depois do 0 e do 1, gpio0 e gpio1 é reservado para a interface UART para definir RX e TX
```suggestion
      [ "pico:GP2", "keypad1:C4", "green", [ "h0" ] ],
```